src/Common/src/Interop/Windows/Kernel32/Interop.PurgeComm.cs

```diff
@@ -9,6 +9,14 @@ internal partial class Interop
 {
     internal partial class Kernel32
     {
+        internal static class PurgeFlags
+        {
+            internal const int PURGE_TXABORT = 0x0001;  // Kill the pending/current writes to the comm port.
```

Comment:
```suggestion
            internal const uint PURGE_TXABORT = 0x0001;  // Kill the pending/current writes to the comm port.
```
To match the native argument? (Though it's a const and the compiler will cast it).